.travis.yml

```diff
@@ -67,10 +66,6 @@ before_script:
 
 script:
     - composer validate --strict
-    - vendor/bin/phpstan analyse -c phpstan.neon -l max src/
-
-    - vendor/bin/phpunit
-    - vendor/bin/phpspec run
```

Comment:
We removed it to pass tests? ;>

Reply:
As above - we don't have specs tests defined in plugin.